### PR TITLE
feat: 🎸 Cloud Computing Page

### PR DESCRIPTION
**Pull request by AlexDuchnowski**
Creation of new cloud computing page explaining CCV connections to AWS
and GCP resources

### Pull Request
---

#### PR Summary
Briefly describe the changes.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [x] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [ ] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [x] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

File: content/english/services/infrastructure/cloud-computing.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+tagTitle: Cloud Computing (AWS & GCP) - Center for Computation and Visualization
+tagDescription: Looking for cloud computing resources for your AI/ML project? Check out AWS and GCP resources.
+
+date: 2019-01-22T18:29:23.000+00:00
+category: Computing
+icon: server2
+lead: "Resources for cloud computing and AI/ML programs (Amazon Web Services and Google Cloud Platform)"
+haas: true
+extra:
+  text: "Risk Classification: N/A (Not owned by Brown University)"
```
Comment (mcmcgrath13) [inline content/english/services/infrastructure/cloud-computing.md:13]: does this mean any usage of these services needs to reviewed by security? if you have risk category 2 data, can you use it here?

Comment (mcmcgrath13) [reply content/english/services/infrastructure/cloud-computing.md:13]: @mttunca

File: content/english/services/infrastructure/cloud-computing.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+[Cloud Credits for Research](https://aws.amazon.com/grants/) are available to all Brown faculty members.
+
+Dozens of datasets are made publicly available to the global research community at no cost. Some examples include:
+
+* [Open NeuroData](https://registry.opendata.aws/open-neurodata/)
+* [Kids First](https://registry.opendata.aws/kids-first/)
+* [Landsat 8](https://registry.opendata.aws/landsat-8/)
+* [MUR](https://registry.opendata.aws/mur/)
+
+For more information about Machine Learning with AWS, go [here](https://aws.amazon.com/machine-learning/).
```
Comment (mcmcgrath13) [inline content/english/services/infrastructure/cloud-computing.md]: A `here` link is not very accessible as it doesn't tell someone using a screen reader where they are going.  Could you make that more descriptive.  Maybe something like `Go to [AWS Machine Learning] for more information.`?

File: content/english/services/infrastructure/cloud-computing.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+(COMING SOON): [STRIDES](https://datascience.nih.gov/strides) agreement
+
+# Google Cloud Platform
```
Comment (mcmcgrath13) [inline content/english/services/infrastructure/cloud-computing.md:33]: What about general use of GCP for research?  Is there a process for requesting a project or management of a billing account you can use?  I know this is currently a pain point that I believe @mttunca had said was being worked on.

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]

File: content/english/services/infrastructure/cloud-computing.md (hunk 2 above)
Comment (AlexDuchnowski) [reply content/english/services/infrastructure/cloud-computing.md]: I just changed the link name. Is it good now?

**Comment by AlexDuchnowski** [comment]
@mcmcgrath13 By the way, more information on cloud computing is being collected, so please don't merge the PR yet (even if the conversations about it so far have been resolved). Thanks.

File: content/english/services/infrastructure/cloud-computing.md (hunk 2 above)
Comment (mcmcgrath13) [reply content/english/services/infrastructure/cloud-computing.md]: looks great! thanks!

**Comment by yangliu2009** [comment]
Closing the pull request for now, as there are discussions ongoing for this.

**Pull request closed without merging**